.storybook/Colors.js

```diff
@@ -12,34 +12,33 @@ const legacy = Object.keys(legacyTheme.colors)
   .map(key => ({ key, value: legacyTheme.colors[key] }))
   .filter(color => !Array.isArray(color.value))
 
-const Chip = props => <Box width={1} px={5} py={4} bg={props.color} />
+const Chip = props => props.color ?
```

Comment:
When checking with `Standard` linter, this line is being warned:
```
/design-system/.storybook/Colors.js:15:36: '?' should be placed at the beginning of the line.
```

Reply:
Additional linter warnings also picked up these, which code being linted may be outside of the scope of this PR. But thought that I'd share, if you'd like to simply update the code format along in this PR:
```
/design-system/.storybook/Colors.js:3:8: 'styled' is defined but never used.
/design-system/.storybook/Colors.js:22:19: Infix operators must be spaced.
/design-system/.storybook/Colors.js:22:24: Infix operators must be spaced.
/design-system/.storybook/Colors.js:22:29: Infix operators must be spaced.
/design-system/.storybook/Colors.js:33:14: Infix operators must be spaced.
/design-system/.storybook/Colors.js:33:19: Infix operators must be spaced.
/design-system/.storybook/Colors.js:33:24: Infix operators must be spaced.
/design-system/.storybook/Colors.js:84:5: Newline required at end of file but not found.
```

Reply:
Will make sure to run the linter before I push!

Reply:
Should probably go ahead and set up [prettier](https://github.com/prettier/prettier) in this repo. Any objections to that?

Reply:
Prettier tool would be great. 👍🏻